modules/nf-core/miniprot/align/main.nf

```diff
@@ -0,0 +1,39 @@
+process MINIPROT_ALIGN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::miniprot=0.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/miniprot:0.5--h7132678_0':
+        'quay.io/biocontainers/miniprot:0.5--h7132678_0' }"
+
+    input:
+    tuple val(meta), path(pep)
+    path ref
```

Comment:
Can you add a meta to the index cfr most other aligners?

Reply:
👍🏼